reg.py

```diff
@@ -50,23 +64,26 @@ def read_dataset():
 
 
 
-# Model Creation
-def multilayer_perceptron(x, weights, biases):
```

Comment:
We talked about using Slim layers. Please switch to it.

Reply:
Ya. I will switch to Slim. Initially, I wanted to get some practice on raw_tensorflow. So now I can change it to Slim.